python/datasources/cdc_wisqars.py

```diff
@@ -122,6 +115,9 @@ def write_to_bq(self, dataset, gcs_bucket, **attrs):
             table_name = f"{demographic}_{geo_level}_{table_type}"
 
             if table_type == CURRENT:
+                df_for_bq[std_col.TIME_PERIOD_COL] = (
+                    df_for_bq[std_col.TIME_PERIOD_COL] + '-01'
+                )
```

Comment:
what is this doing?

Reply:
This adds a month to the code. It seems that the `preserve_only_current_time_period_rows` function requires a month to parse it correctly: `(constant) DT_FORMAT_YYYY_MM: Literal['%Y-%m']`